tests/annotation/approval/src/main/java/demo/Application.java

```diff
@@ -80,7 +80,10 @@ public void configure(ClientDetailsServiceConfigurer clients) throws Exception {
 		            .authorities("ROLE_CLIENT")
 		            .scopes("read")
 		            .resourceIds("oauth2-resource")
-		            .secret("secret");
+		            .secret("secret")
+		    .and()
+		        .withClient("my-client-with-unlimited-scopes")
```

Comment:
Can you rename the `client-id` to `my-client-with-empty-scopes`. Please apply to all instances.

Reply:
Done.